src/pages/Bridge/Tabs.tsx

```diff
@@ -62,7 +71,7 @@ export const Tabs = ({
           setTxsFilter(BridgeTxsFilter.NONE)
           setActiveTab(BridgeTab.HISTORY)
         }}
-        className={activeTab === 'history' ? 'active' : ''}
+        className={activeTab === BridgeTab.HISTORY ? 'active' : ''}
```

Comment:
```suggestion
        className={activeTab === BridgeTab.HISTORY && 'active'}
```

Reply:
The suggested change results in:
```
<div class="false"/>
```

in the DOM. The short circuit stops and returns `false` when `activeTab === BridgeTab.HISTORY` is `false`